### DISCUSSION TITLE
pastefy.ga stats. >200,000 unique visitors/month & more 🥳📈 #pastefyreport

### DISCUSSION THREAD

**Discussion by JulianFun123** [interaapps/pastefy#48] [General]
Pastefy started 5 years as a simple tool embedded into the InteraApps homepage. Now it became the biggest project we made with currently 165 stars on Github and over 700 users registered. 

In the last 30 Days (From 6. April to 6. May 2023) we had over 2 Million requests to our Pastefy server and 216.000 unique visitors (Information by Cloudflare). 
70% of these requests were raw requests, 17% Api calls and the rest frontend ressources like javascript or css.

Our database has 12,893 pastes and 1,619 of them have been created by logged in users. In April alone 1200 Pastes have been created.

Pastefy is constantly growing in usage, by users and by requests. Thank you for using Pastefy.

![image](https://user-images.githubusercontent.com/42511596/236591895-16224b42-1954-4cc2-8ee0-24178c6a5a55.png)

**Comment by PhilNehring** [comment]
Macher
